.github/workflows/release_dev.yml

```
@@ -110,7 +111,7 @@ jobs:
         uses: softprops/action-gh-release@v2.0.9
         with:
           name: ChapterMaster ${{ needs.gamemaker_build.outputs.suffix }}
-          tag_name: dev/${{ github.ref_name }}
+          tag_name: ${{ steps.tag_info.outputs.tag_name }}
```

actionlint (1.7.4)

property "tag_info" is not defined in object type {}

(expression)